LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -45,10 +45,10 @@ public LdapAuthenticationProviderPlugin(IApplicationHost applicationHost, ILogge
         {
             _logger = logger;
             _applicationHost = applicationHost;
-            _httpClient = new(() => httpClientFactory.CreateClient());
+            _httpClientFactory = httpClientFactory;
         }
 
-        private HttpClient HttpClient => _httpClient.Value;
+        private HttpClient HttpClient => _httpClientFactory.CreateClient();
```

Comment:
```suggestion
        private HttpClient HttpClient => _httpClientFactory.CreateClient(NamedClient.Default);
```

Might need a new `using` statement at the top

Reply:
Added it. Did need a using statement.